.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - run: sudo apt update
       - name: "Install packages"
         run: sudo apt-get install g++-9 libsdl1.2-dev libsdl-net1.2-dev libsdl-sound1.2-dev
-      - name: Build
+      - name: "Build"
```

Comment:
are quotes needed in the YAML value? Unquoted is valid YAML, but maybe GitHub's workflow tool requires it? Just curious!

Reply:
No, they're not required - but earlier I had an issue where GitHub mangled name in one of yaml fields (I don't remember which one though, it's probably fixed already). So I started quoting name fields and liked it, because vim highlights them as strings, so they stand out from the commands.

In this commit I make the quoting consistent across all the jobs defined in this file.

Reply:
Ahh, makes sense, especially if it keeps your editor treating it like one string.